rasa/telemetry.py

```diff
@@ -563,6 +563,27 @@ def toggle_telemetry_reporting(is_enabled: bool) -> None:
 
     rasa_utils.write_global_config_value(CONFIG_FILE_TELEMETRY_KEY, configuration)
 
+def filter_errors(event: Dict[Text, Any], _unused_hint: Optional[Dict[Text, Any]] = None) -> None:
```

Comment:
Two things here need amending:

- please rename function to either `filter_import_errors` or `filter_tf_errors`
- return type is actually `Optional[Dict[Text, Any]]`

Reply:
Kept it as filter_errors going forward we can utilize this function if any other errors that we need to handle. What are your thoughts on it?

Reply:
Yes, true, better not to pin the name to a specific error. However, since the type of error that would be handled here would always be usage errors, I'd still rename this to `filter_usage_errors`.